Documentation/gettingstarted/host-firewall.rst

```diff
@@ -22,24 +22,45 @@ Enable the Host Firewall in Cilium
 
 Deploy Cilium release via Helm:
 
-.. parsed-literal ::
+.. code:: shell-session
 
-    helm install cilium |CHART_RELEASE|        \\
+    $ helm install cilium |CHART_RELEASE|      \\
       --namespace kube-system                  \\
       --set hostFirewall=true                  \\
-      --set devices=ethX,ethY
+      --set devices='{ethX,ethY}'
+
+The ``devices`` refer to the network devices Cilium is configured on such as
+``eth0``. Omitting this option leads Cilium to auto-detect what interfaces the
+host firewall applies to.
 
 At this point, the Cilium-managed nodes are ready to enforce network policies.
 
+.. note::
+
+    The host firewall is not compatible with per-endpoint routing. This option
+    is enabled by default on managed services (AKS, EKS, GKE), so in order to
+    use the host firewall on those environments, per-endpoint routing must be
+    disabled. For example, on GKE, replace ``--set gke.enabled=true`` with
+    ``--set ipam.mode=kubernetes --set endpointRoutes.enabled=false --set
+    tunnel=disabled``.
+
+    See also `GitHub issue #13121
+    <https://github.com/cilium/cilium/issues/13121>`_.
+
 
 Attach a Label to the Node
 ==========================
 
+.. note::
+
+    This must be done *after* deploying Cilium, or the agent may not pick up
+    the change.
```

Comment:
I'm not sure I would add a note for that bug. We'll likely get it fixed shortly and if folks follow the guide, they shouldn't run into this issue anyway.

Reply:
Wasn't 100% sure either... Ok, I can remove it.